src/js/components/AppStatusComponent.jsx

```diff
@@ -59,11 +59,8 @@ var AppStatusComponent = React.createClass({
   render: function () {
     var model = this.props.model;
 
-    var statusClassSet = classNames({
-      "app-status": true,
-      "text-warning": this.isWarningStatus(),
-      "text-danger": model.status === AppStatus.DELAYED
-    });
+    var statusClassSet = classNames("app-status",
+      statusNameMapping[model.status].toLowerCase());
```

Comment:
Yeah, good point! What about introducing another mapping, to have a little more descriptive class names?